contracts/NFTStorefront.cdc

```diff
@@ -59,9 +59,12 @@ pub contract NFTStorefront {
         storefrontAddress: Address,
         listingResourceID: UInt64,
         nftType: Type,
+        nftUUID: UInt64, 
         nftID: UInt64,
         ftVaultType: Type,
-        price: UFix64
+        price: UFix64,
+        customID: String,
```

Comment:
I know this has potential to be problematic for storage, but would we consider this being a map so you could have more than one customID? And be able to label it? Eg. like `metadata: {String:String}` (we have this in our NFT contracts for TopShot and AllDay)

Reply:
You can use JSON here and store whatever you'd like in it. The only limitation would be the length of a `String` in Cadence.

Reply:
Ok yeah! Makes sense - is there a preference for String(JSON) over using the `{String:String}`? I suppose it may be more flexible for different inputs

Reply:
curious - what are some example use-cases for customID?